api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/HivQueries.java

```diff
@@ -0,0 +1,113 @@
+package org.openmrs.module.eptsreports.reporting.library.queries;
+
+public class HivQueries {
+
+  /**
+   * Patients on TB Treatment
+   *
+   * @param adultoSeguimentoEncounterTypeId
+   * @param arvPediatriaSeguimentoEncounterTypeId
+   * @param tbStartDateConceptId
+   * @param tbEndDateConceptId
+   * @param tbProgramId
+   * @param patientStateId
+   * @param activeTBConceptId
+   * @param yesConceptId
+   * @param tbTreatmentPlanConceptId
+   * @param startDrugsConceptId
+   * @param continueRegimenConceptId
+   * @return
+   */
+  public static String getPatientsOnTbTreatmentQuery(
```

Comment:
It looks like this  query  is same as the one at  `TbQueries`...it would be better if remove one of them. we may use the from TB

Reply:
Made the relevant changes. Thanks